src/system_tray.h

```diff
@@ -28,3 +31,4 @@ namespace system_tray {
   end_tray();
 
 }  // namespace system_tray
+#endif // SUNSHINE_SYSTEM_TRAY_H
```

Comment:
```suggestion
#endif  // SUNSHINE_SYSTEM_TRAY_H
```

This will resolve the lint error.